packages/app-project/src/screens/ProjectHomePage/components/Hero/Hero.stories.js

```diff
@@ -91,16 +90,14 @@ const snapshot = {
   project: PROJECT,
   user: USER
 }
-const store = Store.create(snapshot)  
+const store = Store.create(snapshot)
 
 export function Default({ isWide }) {
   return (
     <RouterMock>
-      <MediaContextProvider>
```

Comment:
<Media> hasn't been used in Hero for at least a year, so this was just lingering in its Storybook file.